common/src/paths.rs

```diff
@@ -0,0 +1,22 @@
+use std::path::PathBuf;
+
+pub const QUALIFIER: &'static str = "org";
+pub const ORGANIZATION: &'static str = "holochain";
+pub const APPLICATION: &'static str = "holochain";
+pub const KEYS_DIRECTORY: &'static str = "keys";
+
+/// Returns the path to the root config directory for all of Holochain.
+/// If we can get a user directory it will be an XDG compliant path
+/// like "/home/peter/.config/holochain".
+/// If it can't get a user directory it will default to "/etc/holochain".
+pub fn config_root() -> PathBuf {
+    directories::ProjectDirs::from(QUALIFIER, ORGANIZATION, APPLICATION)
+        .map(|dirs| dirs.config_dir().to_owned())
+        .unwrap_or_else(|| PathBuf::new().join("/etc").join(APPLICATION))
```

Comment:
```suggestion
        .unwrap_or_else(|| PathBuf::from("/etc").join(APPLICATION))
```
Also I still think, this is a bad idea

Reply:
What is a bad idea?

Reply:
Btw, proof, that these are the same:
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=978badfa745d4fa9b744dbf8229b28e9